examples/pytorch/distributed_stl10/README.md

```diff
@@ -0,0 +1,6 @@
+# PyTorch STL10 example script
+
+You can launch the job (two single GPU nodes) with:
+```
+sbatch jean-zay-doc/examples/pytorch/distributed_stl10/pytorch_distributed_stl10_example.sh
```

Comment:
I propose this change:
```suggestion
cd jean-zay-doc/examples/pytorch/distributed_stl10/
sbatch ./pytorch_distributed_stl10_example.sh
```
And also the hard-coded folder in the `pytorch_distributed_stl10_example.sh` script (see next comment). This will avoid silly mistakes if someone clone the repository in a different place from  `$WORK`.